port/common/omrsysinfo.c

```diff
@@ -515,9 +515,10 @@ omrsysinfo_get_load_average(struct OMRPortLibrary *portLibrary, struct J9PortSys
  * The cpuTime and timestamp values have no absolute significance: they should be used only to compute
  * differences from previous values.
  * On an N-processor  system, cpuTimeStats.cpuTime may increase up to N times faster than real time.
+ * On z/OS use cpuLoad to hold real cpu for the system. Other platform should default to cpuLoad -1.
```

Comment:
We should not mention platform specifics in the common port library documentation. The point of the common APIs is to abstract platform specifics. As mentioned in the previous review comment this detail should move up to the definition point of `cpuLoad` and `cpuTime`. The documentation here should be improved however:

```
/**
 * Obtain the cumulative CPU utilization of all CPUs on the system.
 *
 * The cumulative CPU utilization can be derived either from the cpuLoad field (if computed) directly or
 * the combination of cpuTime and timestamp values (if computed) by taking differences from previous 
 * values.
 *
 * Note that cpuLoad and cpuTime are mutually exclusive and only one of them will be present. The other 
 * will have all value of -1 signifying that the value could not nr computed.
```